README.md

```diff
@@ -1 +1,145 @@
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Math Magicians <a name="about-project"></a>
+
+**Math Magicians** is a website for all fans of mathematics which can make simple mathematical calculations and you can read a random math-related quote.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Calculator App**
+- **Single page application**
+- **Automated quotes generator**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Nodejs
+- A text editor of your choice
+- A properly working pc or laptop
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/naolkidanu/calculator.git
+```
+
+### Install
+
+Install this project with node packages:
+
+```sh
+  cd my-project
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm jest
+```
+
+### Deployment
+
+You can deploy this project using: GitHub but if you want to deploy make sure you let me know first :3
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Naol Kidanu Merdasa**
+
+- GitHub: [Naol Kidanu Merdasa](https://github.com/naolkidanu/calculator)
+- Twitter: [Naol Kidanu Merdasa](https://twitter.com/naolkidanu)
+- LinkedIn: [Naol Kidanu Merdasa](https://linkedin.com/in/naolkidanu)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Deployment**
+- [ ] **Add Calculator**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please, share it with your friends.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for letting me build this awesome project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](MIT.md) licensed.
```

Comment:
- [ ] As stated by the previous reviewer, the MIT license link is still broken and i notice the file path you are trying to target does not exist in your project root directory. Kindly add the MIT.md file to your root directory so that the file path targeted in your readme file can be directed to it

Reply:
Hi @brainconnect93 
I appreciate your feedback and will address it.